README.md

```diff
@@ -3,7 +3,7 @@
 
 ### Spirit Design System
 
-[![Maintained with Lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org)
+[![Maintained with Lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org) [![Code Quality Checks](https://github.com/lmc-eu/spirit-design-system/actions/workflows/test.yaml/badge.svg?branch=main)](https://github.com/lmc-eu/spirit-design-system/actions)
```

Comment:
Can we have the new badge on a separate line? It still gets rendered in one row anyway.

Reply:
Ok, fixed.